main.py

```diff
@@ -1,7 +1,38 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+
+    new_genres = ["Western", "Action", "Dramma"]
+    for genre in new_genres:
+        Genre.objects.create(name=genre)
+
+    new_actors = ["George Klooney", "Kianu Reaves",
```

Comment:
It would be better to store a list of tuples where first element would be first name and second element last name. Then you will not need to call split() method and just use tuple unpacking which would be more pythonic` approach.
Also I would suggest to place each element of the list from new line